packages/rich-text/src/SdkProvider.tsx

```diff
@@ -8,7 +8,7 @@ interface SdkProviderProps {
 }
 
 function useSdk({ sdk }: SdkProviderProps) {
-  const sdkMemo = React.useMemo<FieldAppSDK>(() => sdk, [sdk.parameters.instance.activeLocales]); // eslint-disable-line -- TODO: explain this disable
+  const sdkMemo = React.useMemo<FieldAppSDK>(() => sdk, []); // eslint-disable-line -- TODO: explain this disable
```

Comment:
`TODO: explain this disable` I guess the reason for this is we don't want to trigger a load of rerenders and that it's okay to just create the memo when this hook gets mounted and the side effects are managed by the even listeners in each of the sdk properties themselves